Aptoide/aptoide-client-v8
# Feature/an 1711 post cards layout

What does this PR do?

   This pull-request parses Post In Timeline cards and shows them in the Timeline with different layouts. There are 3 types of post (Article/Video/App). 

Where should the reviewer start?

- [ ] SocialPostMediaViewHolder - timeline_social_post_media_item.xml   (article+video)
- [ ] SocialPostRecommendation - timeline_social_post_recommendation_item.xml (app)

How should this be manually tested?

  Open Aptoide v8 > Stores > ana2016 > Timeline > Check for Post

What are the relevant tickets?

Tickets related to this pull-request: [AN-1711](https://aptoide.atlassian.net/browse/AN-1711)

Screenshots (if appropriate):
N/A

Questions:

   Is there a blog post? N/A
   Does this add new dependencies which need to be added? No
